### PR TITLE
adding booleans concept

### PR DESCRIPTION
**Pull request by rlmark**
Adding booleans concept content. 

Listing as WIP for now.

### CODE REVIEW

**Comment by rlmark** [comment]
This one is probably the next logical step after `Basics` as a concept.

File: exercises/concept/pacman-rules/.docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+In this exercise, you need to translate some rules from the classic game Pac-Man into Elixir functions.
```
Comment (ErikSchierboom) [inline exercises/concept/pacman-rules/.docs/instructions.md:3]:
```suggestion
In this exercise, you need to translate some rules from the classic game Pac-Man into Unison functions.
```

File: exercises/concept/pacman-rules/.docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+> false && (base.bug "oh no")
+  ⧩
+  false
+```
+
+```unison
+> true || (base.bug "oh no")
+  ⧩
+  true
+```
```
Comment (ErikSchierboom) [inline exercises/concept/pacman-rules/.docs/introduction.md:36]: Do students need this information to solve the exercise? If not, could you remove it here (it is perfect for the `about.md` file).

**Review by ErikSchierboom — APPROVED** [review]
Two tiny nits.

**Pull request merged**